apps/builddao/widget/feed/post/post-creator.jsx

```diff
@@ -3,7 +3,50 @@ const [postContent, setPostContent] = useState("What do you have in mind?");
 const [hideAdvanced, setHideAdvanced] = useState(true);
 const [labels, setLabels] = useState([]);
 
-const name = Social.get(`${context.accountId}/profile/name`);
+function generateUID() {
+  const maxHex = 0xffffffff;
+  const randomNumber = Math.floor(Math.random() * maxHex);
+  return randomNumber.toString(16).padStart(8, '0');
+}
+
+const postToCustomFeed = ({ feed, text, labels }) => {
+  const postId = generateUID()
+  if (!labels) labels = []
+  return Social.set({
+    "update": {
+      [postId]: {
+        "content": JSON.stringify({
+          type: "md",
+          text,
+          labels,
+        }),
+        "metadata": {
+          type: feed
+        },
+      },
+    },
+    "post": {
+      "main": JSON.stringify({
+        type: "md",
+        text: `[EMBED](${context.accountId}/${feed}/${postId})`,
+      }),
+    },
+    "index": {
+      "post": JSON.stringify([
+        { key: "main", value: { type: "md" }},
+        { key: feed, value: { type: "md" }},
```

Comment:
I think we should consider changing this to it is not post/{feed}, but every/{feed}